src/olympia/blocklist/mlbf.py

```diff
@@ -87,10 +87,11 @@ def generate_mlbf(cls, stats, blocked, not_blocked):
         cascade.set_crlite_error_rates(
             include_len=error_rates[0], exclude_len=error_rates[1])
 
-        cascade.initialize(include=blocked, exclude=not_blocked)
-
         stats['mlbf_blocked_count'] = len(blocked)
         stats['mlbf_notblocked_count'] = len(not_blocked)
+
+        cascade.initialize(include=blocked, exclude=not_blocked)
```

Comment:
Does putting the line here change anything?

Reply:
It just means the stats dict is populated earlier. When i had the traceback on sentry i wished i had that information